contracts/farm/LiquidityMiningV1.sol

```diff
@@ -129,8 +134,8 @@ contract LiquidityMiningV1 is ILiquidityMining, LiquidityMiningStorageV1 {
 	// TODO: this should only be used by the LiquidityMiningV2 contract??
 	/// @notice This function finishes the migration process disabling further withdrawals and claims
 	/// @dev migration grace period should have started before this function is called.
-	function finishMigrationGracePeriod() external onlyAuthorized onlyBeforeMigrationGracePeriodFinished {
-		require(migrationGracePeriodState == MigrationGracePeriodStates.Started, "Migration hasn't started yet");
+	function finishMigrationGracePeriod() external onlyAuthorized {
+		require(migrationGracePeriodState >= MigrationGracePeriodStates.Started, "Migration hasn't started yet");
```

Comment:
I'm not sure about this change ... in which case we might need the `>=` ?

Reply:
this change is because its allowed to migrate funds, pools and users in any order, so finishMigrationGracePeriod() is called more than once